<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllSetting</class>
 <widget class="QWidget" name="AllSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>工作目录</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>351</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>参量测量选项</string>
   </property>
   <widget class="QCheckBox" name="memCBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>内存</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值 （M）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="memLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpuLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值 （%）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="batteryLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="batteryCBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电池温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值（℃）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpuTempLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuTempCBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值（℃）：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="wifiCBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wifi流量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值（M）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobileLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="mobileCBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mobile流量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值（M）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wifiLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>安卓安装路径</string>
   </property>
   <widget class="QLineEdit" name="AndroidLlineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="androidBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>收件邮箱设置</string>
   </property>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
